src/main/java/uk/gov/ons/ssdc/caseprocessor/model/entity/UacQidLink.java

```diff
@@ -17,15 +17,14 @@
 public class UacQidLink {
   @Id private UUID id;
 
-  @Column(name = "qid")
```

Comment:
this was here because otherwise the index (which has admittedly been deleted) won't work because of some strange JPA/Hibernate quirk

Reply:
Ah, I didn't know that. I'll put it back with a comment 👍